src/App.tsx

```diff
@@ -1,61 +1,116 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+
+function getUser(userId: number): User | null {
```

Comment:
![image](https://user-images.githubusercontent.com/91411692/233009687-55e3bd6d-0e4f-4b4a-8ddb-7a1ae06cf2de.png)
```suggestion
function getUserById(id: number): User | null {
```